docs/agenda.md

```diff
@@ -23,60 +23,79 @@ Bi-weekly DIF call notes on DWN specifications.
 - Andor Kesselman @andorsk email: andor@benri.io
 - Liran Cohen @lirancohen
 
+## DIF Meeting March 19, 2023
```

Comment:
I'm fairly certain this section was meant to be dated today.
```suggestion
## DIF Meeting 2023-04-19
```

Reply:
You are correct @TallTed! Thanks!